elixir/test/contracts_test.exs

```diff
@@ -200,5 +250,65 @@ defmodule Trento.ContractsTest do
 
       assert cloudevent_expiration == DateTime.add(cloudevent_time, 5, :minute)
     end
+
+    test "should encode with additional attributes" do
+      event = %Test.Event{id: UUID.uuid4()}
+      cloudevent_id = UUID.uuid4()
+      time = DateTime.utc_now()
+      expiration = DateTime.add(time, 3, :minute)
+      time_attr = %Google.Protobuf.Timestamp{seconds: time |> DateTime.to_unix()}
+      expiration_attr = %Google.Protobuf.Timestamp{seconds: expiration |> DateTime.to_unix()}
+      user_id = 1
+
+      cloudevent = %CloudEvent{
```

Comment:
nitpick: we could maybe use the `build_cloud_event` here

Reply:
thanks, overlooked 1d3c6e762f97cb48df2b32bb63909fac8ce8aae1